src/main/java/subway/dto/RouteResponse.java

```diff
@@ -0,0 +1,24 @@
+package subway.dto;
+
+import java.util.List;
+
+public class RouteResponse {
+    private List<String> station;
+    private int fee;
+
+    public RouteResponse() {
```

Comment:
```suggestion
    private RouteResponse() {
```

기본 생성자는 private으로 막아두는 건 어떨까요?

Reply:
직렬화의 문제로 기본생성자를 생성하고 있었는데 reflaction을 통해 가져오기 때문에 private으로 막아도 가져올수있군요. 확실히 privated으로 막아서 이를 해결할 수 있겠군요...